api/cpp/include/slint.h

```diff
@@ -265,6 +265,9 @@ class WindowAdapterRc
         }
     }
 
+    /// \private
+    const cbindgen_private::WindowAdapterRcOpaque &handle() { return inner; }
```

Comment:
😭
Is there not a way to avoid adding that?

Reply:
I don't think so. But... the class is not public API :) (it's accessible via hacks though).

Reply:
I don't think so. But... the class is not public API :) (it's accessible via hacks though).